pkgs/shells/loksh/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, meson
+, ninja
+, pkg-config
+, ncurses
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "loksh";
+  version = "6.9";
+
+  src = fetchFromGitHub {
+    owner = "dimkr";
+    repo = pname;
+    rev = "${version}";
```

https://nixos.org/manual/nixpkgs/unstable/#sec-syntax

```suggestion
    rev = version;
```

---

thanks for the suggestion. made the change.